src/ComScoreStreamingAnalytics.ts

```diff
@@ -82,10 +90,43 @@ export class ComScoreStreamingAnalytics {
     this.player.on(this.player.exports.PlayerEvent.StallStarted, this.stallStarted);
     this.player.on(this.player.exports.PlayerEvent.StallEnded, this.stallEnded);
     this.player.on(this.player.exports.PlayerEvent.AdBreakStarted, this.adBreakStarted);
+    this.player.on(this.player.exports.PlayerEvent.TimeChanged, this.timeChanged);
+  }
+
+  private blured = () => {
```

Comment:
Maybe rename to some meaning full method name. e.g. `enteredBackground` or something like that